src/components/calcite-split-button/calcite-split-button.scss

```diff
@@ -1,3 +1,7 @@
+:host {
+  @apply inline-block w-auto align-middle;
```

Comment:
Copied this from `calcite-button`. is this correct @eriklharper?

Reply:
Should the non-display styles be set on the container instead?

Reply:
That looks right since the native `<button>` is also `inline-block` by default.

Reply:
> Should the non-display styles be set on the container instead?

Good question.  I think the display should probably be set on the host element, but as far as the other properties, there's probably a lot you can move to a container div, unless the host needs like a position property or some other parent-child css property that enables styles to work for elements underneath the host element, like flex for example.

Reply:
I'll remove the width and align stuff.